public/scripts/socket-github.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Client-side Event Handling for deploy/github
+ */
+$(function () {
+  var socket = io();
+  socket.emit('ghpages-deploy', {
+    'email': "#{email}",
```

Comment:
i tried and it doesn't. However, i found something that might solve this http://feather.elektrum.org/book/src.html